cmd/initializer/main.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"github.com/pingcap/tidb-operator/pkg/initializer"
+	"github.com/pingcap/tidb-operator/pkg/version"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/component-base/logs"
+	"k8s.io/klog"
+	"os"
+)
+
+var (
+	printVersion     bool
+	namespace        string
+	component        string
+	podName          string
+	verifyPeriodDays int
+)
+
+func init() {
+	flag.BoolVar(&printVersion, "V", false, "Show version and quit")
+	flag.BoolVar(&printVersion, "version", false, "Show version and quit")
+	flag.StringVar(&component, "component", "all", "The component to initializer resource for")
```

Comment:
Better to be a string array, that is, able to specify multiple components but not all

Reply:
fix